### ISSUE TITLE
New submissions for Friday, 2 August 2024

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#358]
# New submissions for Friday, 2 August 2024
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'star-forming', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', ' AGN ']

#### Lower-mass-gap Black Holes in Dense Star Clusters
 - **Authors:** Claire S. Ye (CITA), Kyle Kremer, Scott M. Ransom, Frederic A. Rasio
 - **Subjects:** Subjects:
High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2408.00076

 - **Pdf link:** https://arxiv.org/pdf/2408.00076

 - **Abstract**
 The existence of compact stellar remnants in the mass range $2-5\,M_{\odot}$ has long been debated. This so-called lower mass gap was initially suggested by the lack of low-mass X-ray binary observations with accretors in this mass range, but it has recently been called into question following newer observations, including a lower-mass-gap candidate with a millisecond pulsar companion in the dense globular cluster NGC~1851. Here we model NGC~1851 with a grid of similar dense star clusters utilizing the state-of-the-art Monte Carlo $N$-body code \texttt{CMC}, and we specifically study the formation of lower-mass-gap black holes. We demonstrate that both massive star evolution and dynamical interactions can contribute to forming lower-mass-gap black holes. In general, the collapse of massive remnants formed through mergers of neutron stars or massive white dwarfs produces the largest number of lower-mass-gap black holes among all formation channels. However, in more massive clusters, supernova core collapse can contribute comparable numbers. Our NGC~1851-like models can reproduce millisecond pulsar -- lower-mass-gap black hole binaries similar to the observed system. Additionally, the lower-mass-gap black holes can also become components of dynamically assembled binaries, and some will be in merging black hole--neutron star systems similar to the recently detected gravitational wave source GW230529. However, the corresponding merger rate is probably $\lesssim 1~{\rm Gpc^{-3}\,yr^{-1}}$.
#### Approximating Rayleigh Scattering in Exoplanetary Atmospheres using Physics-informed Neural Networks (PINNs)
 - **Authors:** David Dahlbüdding, Karan Molaverdikhani, Barbara Ercolano, Tommaso Grassi
 - **Subjects:** Subjects:
Earth and Planetary Astrophysics (astro-ph.EP); Instrumentation and Methods for Astrophysics (astro-ph.IM); Machine Learning (cs.LG); Neural and Evolutionary Computing (cs.NE)
 - **Arxiv link:** https://arxiv.org/abs/2408.00084

 - **Pdf link:** https://arxiv.org/pdf/2408.00084

 - **Abstract**
 This research introduces an innovative application of physics-informed neural networks (PINNs) to tackle the intricate challenges of radiative transfer (RT) modeling in exoplanetary atmospheres, with a special focus on efficiently handling scattering phenomena. Traditional RT models often simplify scattering as absorption, leading to inaccuracies. Our approach utilizes PINNs, noted for their ability to incorporate the governing differential equations of RT directly into their loss function, thus offering a more precise yet potentially fast modeling technique. The core of our method involves the development of a parameterized PINN tailored for a modified RT equation, enhancing its adaptability to various atmospheric scenarios. We focus on RT in transiting exoplanet atmospheres using a simplified 1D isothermal model with pressure-dependent coefficients for absorption and Rayleigh scattering. In scenarios of pure absorption, the PINN demonstrates its effectiveness in predicting transmission spectra for diverse absorption profiles. For Rayleigh scattering, the network successfully computes the RT equation, addressing both direct and diffuse stellar light components. While our preliminary results with simplified models are promising, indicating the potential of PINNs in improving RT calculations, we acknowledge the errors stemming from our approximations as well as the challenges in applying this technique to more complex atmospheric conditions. Specifically, extending our approach to atmospheres with intricate temperature-pressure profiles and varying scattering properties, such as those introduced by clouds and hazes, remains a significant area for future development.
#### Dust-Gas Coupling in Turbulence- and MHD Wind-Driven Protoplanetary Disks: Implications for Rocky Planet Formation
 - **Authors:** Teng Ee Yap, Konstantin Batygin
 - **Subjects:** Subjects:
Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2408.00159

 - **Pdf link:** https://arxiv.org/pdf/2408.00159

 - **Abstract**
 The degree of coupling between dust particles and their surrounding gas in protoplanetary disks is quantified by the dimensionless Stokes number. The Stokes number (St) governs particle size and spatial distributions, in turn establishing the dominant mode of planetary accretion in different disk regions. In this paper, we model the characteristic St of particles across time in disks evolving under both turbulent viscosity and magnetohydrodynamic (MHD) disk winds. In both turbulence- and wind-dominated disks, we find that collisional fragmentation is the limiting mechanism of particle growth, and the water-ice sublimation line constitutes a critical transition point between dust settling, drift, and size regimes. The St dichotomy across the ice-line translates to distinct planet formation pathways between the inner and outer disk. While pebble accretion proceeds slowly for rocky embryos within the ice-line (across most of parameter space), it does so rapidly for volatile-rich embryos beyond it, allowing for the growth of giant planet cores before disk dissipation. Through simulations of rocky planet growth, we evaluate the competition between pebble accretion and classical pairwise collisions between planetesimals. We conclude that the dominance of pebble accretion can only be realized in disks that are driven by MHD winds, slow-evolving, and devoid of pressure maxima that may concentrate solids and give rise of planetesimal rings. Such disks are extremely quiescent, with Shakura-Sunyaev turbulence parameters $\alpha_{\nu} \sim 10^{-4}$. We conclude that for most of parameter space corresponding to values of $\alpha_{\nu}$ reflected in observations of protoplanetary disks ($\gtrsim 10^{-4}$), pairwise collisions constitute the dominant pathway of rocky planet accretion. Our results are discussed in the context of super-Earth origins and Earth's accretion history.
#### MHD lensing in inhomogeneous ISM for qualitative understanding of the morphology of supernova remnants
 - **Authors:** Yoshiaki Sofue
 - **Subjects:** Subjects:
Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2408.00260

 - **Pdf link:** https://arxiv.org/pdf/2408.00260

 - **Abstract**
 Morphological evolution of expanding shells of fast-mode magnetohydrodynamic (MHD) waves through an inhomogeneous ISM is investigated in order to qualitatively understand the complicated morphology of shell-type supernova remnants (SNR). Interstellar clouds with high Alfvén velocity act as concave lenses to diverge the MHD waves, while those with slow Alfvén velocity act as convex lenses to converge the waves to the focal points. By combination of various types of clouds and fluctuations with different Alfvén velocities, sizes, or wavelengths, the MHD-wave shells attain various morphological structures, exhibiting filaments, arcs, loops, holes, and focal strings, mimicking old and deformed SNRs.
#### Stochastic gravitational wave background from cosmological neutrino-dominated accretion flows
 - **Authors:** Yun-Feng Wei, Tong Liu
 - **Subjects:** Subjects:
High Energy Astrophysical Phenomena (astro-ph.HE); General Relativity and Quantum Cosmology (gr-qc)
 - **Arxiv link:** https://arxiv.org/abs/2408.00546

 - **Pdf link:** https://arxiv.org/pdf/2408.00546

 - **Abstract**
 We investigate the stochastic gravitational wave background (SGWB) from neutrino-dominated accretion flows (NDAFs) based on the results of our fallback core-collapse supernova (CCSN) simulations. We find that the predicted SGWB is mainly determined by the typical CCSN initial explosion energy and progenitor metallicity. For the optimistic cases in which the typical initial explosion energy is low, the SGWB from NDAFs without disk outflows might be detected by next-generation space-based interferometers such as Decihertz Interferometer Gravitational wave Observatory (DECIGO) and Big Bang Observer (BBO). In the low-frequency regime $\sim10^{-3}-10^{-1}$ Hz, this background is comparable to that expected from standard inflationary models. Therefore, the SGWB from NDAFs may become a foreground for searches of the SGWB generated in the inflationary epoch. Combining the diffuse NDAF neutrino background and SGWB from NDAFs, one may constrain the properties of the CCSNe and NDAFs.
#### Neglected Silicon Dioxide Polymorphs as Clouds in Substellar Atmospheres
 - **Authors:** Sarah E. Moran, Mark S. Marley, Samuel D. Crossley
 - **Subjects:** Subjects:
Earth and Planetary Astrophysics (astro-ph.EP); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2408.00698

 - **Pdf link:** https://arxiv.org/pdf/2408.00698

 - **Abstract**
 Direct mid-infrared signatures of silicate clouds in substellar atmospheres were first detected in Spitzer observations of brown dwarfs, although their existence was previously inferred from near-infrared spectra. With the Mid-Infrared Instrument (MIRI) instrument on JWST, we can now more deeply probe silicate features from 8 to 10 microns, exploring specific particle composition, size, and structure. Recent characterization efforts have led to the identification in particular of silica (silicon dioxide, SiO$_2$) cloud features in brown dwarfs and giant exoplanets. Previous modeling, motivated by chemical equilibrium considerations, has primarily focused on magnesium silicates (forsterite, enstatite), crystalline quartz, and amorphous silica to match observations. Here, we explore the previously neglected possibility that other crystalline structures of silica, i.e. polymorphs, may be more likely to form at the pressure and temperature conditions of substellar upper atmospheres. We show how these polymorphs may be distinguished from each other with current JWST observations. We also explore how such particles could form and be dynamically lofted and sedimented throughout the atmosphere, and implications for the underlying chemical and dynamical processes governing these objects. We ultimately propose that accounting for the distinct opacities arising from the possible crystalline structure of cloud materials may act as a powerful, observable diagnostic tracer of atmospheric conditions, where particle crystallinity records the history of the atmospheric regions through which clouds formed and evolved. Finally, we highlight that high fidelity, accurate laboratory measurements of silica polymorphs are critically needed to draw meaningful conclusions about the identities and structures of clouds in substellar atmospheres.
#### North Polar Spur: gaseous plume(s) from star-forming regions at $\sim$3-5 kpc from Galactic Center?
 - **Authors:** E.Churazov, I.Khabibullin, A.M.Bykov, N.N.Chugai, R.A.Sunyaev, V.P.Utrobin, I.I.Zinchenko
 - **Subjects:** Subjects:
High Energy Astrophysical Phenomena (astro-ph.HE); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2408.00752

 - **Pdf link:** https://arxiv.org/pdf/2408.00752

 - **Abstract**
 We argue that the North Polar Spur (NPS) and many less prominent structures are formed by gaseous metal-rich plumes associated with star-forming regions (SFRs). The SFRs located at the tangent to the 3-5 kpc rings might be particularly relevant to NPS. A multi-temperature mixture of gaseous components and cosmic rays rises above the Galactic disk under the action of their initial momentum and buoyancy. Eventually, the plume velocity becomes equal to that of the ambient gas, which rotates with different angular speed than the stars in the disk. As a result, the plumes acquire characteristic bent shapes. An ad hoc model of plumes' trajectories shows an interesting resemblance to the morphology of structures seen in the radio continuum and X-rays.

by olozhika (Xing Yuchen). 

2024-08-02
